### PR TITLE
ui: fix physical network guest traffic type tab

### PR DESCRIPTION
**Pull request by shwstppr**
### Description

Fixes icon
Fixes object change

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### Screenshots (if appropriate):
Before
Screenshot from 2022-03-11 13-30-16

After
Screenshot from 2022-03-11 13-30-59

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan ui

**Review by rohityadavcloud — APPROVED** [review]

File: ui/src/views/infra/network/IpRangesTabGuest.vue [hunk 1]
```diff
@@ -137,7 +137,7 @@ export default {
     this.fetchData()
   },
   watch: {
-    network (newItem, oldItem) {
+    resource (newItem, oldItem) {
```
Comment (utchoang) [inline ui/src/views/infra/network/IpRangesTabGuest.vue:140]: @davidjumani With Vue3, to watch an Object, you should use deep. Refer: https://vuejs.org/guide/essentials/watchers.html#deep-watchers

Comment (shwstppr) [reply ui/src/views/infra/network/IpRangesTabGuest.vue:140]: @utchoang Doc says `deep` is needed only in cases where trying to run callback on all nested mutations. Even there example runs fine without it and I don't think we want to run `fetchData` here on any property change of `resource`

**Review by utchoang — APPROVED** [review]

**Review by davidjumani — APPROVED** [review]

**Review by weizhouapache — APPROVED** [review]
tested ok.

**Pull request merged**